models/purchase.py

```diff
@@ -56,6 +57,7 @@ class Purchase(BaseModel):
     # Financial FKs
     payment_id = db.Column(db.Integer, db.ForeignKey("payment.id"))
     refund_id = db.Column(db.Integer, db.ForeignKey("refund.id"))
+    refund_request_id = db.Column(db.Integer, db.ForeignKey("refund_request.id"))
```

Comment:
I've probably just overlooked the relevant flask form / sqlalchemy logic, but could I check where this columns is used/reference?

Reply:
It's not actually used (at the moment) but it allows us to link back to the request as well. Given that (in theory) when processing a request an admin could select different tickets to refund this means we can refer back the original request and confirm what was requested